pkg/database/realm.go

```diff
@@ -384,11 +391,18 @@ func (r *Realm) BeforeSave(tx *gorm.DB) error {
 		r.AddError("passwordWarn", "may not be longer than password rotation period")
 	}
 
+	if r.ShortCodeMaxMinutes < 60 || r.ShortCodeMaxMinutes > 120 {
```

Comment:
60 and 120 should be constants or, more ideally, a runtime configuration as an envvar so we don't have to deploy new code.

Reply:
see todo.